src/stormpath.auth.js

```diff
@@ -135,6 +135,7 @@ angular.module('stormpath.auth',['stormpath.CONFIG'])
         op.then(function(){
           $rootScope.$broadcast(STORMPATH_CONFIG.SESSION_END_EVENT);
         },function(response){
+          $rootScope.$broadcast(STORMPATH_CONFIG.SESSION_END_EVENT);
```

Comment:
What type of error would cause this? Isn't it better to handle that specific error? Uncertain if really any error should end the session... (actually, I'm not even certain it ends the session, or how the event is handled... just guessing:))